### PR TITLE
[block_wise w8a8]Add block_wise w8w8 tunning scripts

### PR DESCRIPTION
**Pull request by coolhok**
## Motivation

add block-wise int8 w8a8 quantization auto tuning script.

## Modifications

1. modify tuning_fused_moe_triton.py adapter block-wise int8 w8a8 moe.
2. add tuning_block_wise_fp8.py for tuning block-wise int8 w8a8 gemm. 

### run block-wise int8 w8a8 gemm tuning
```
python tuning_block_wise_w8a8.py -tp 8
```

### other
When running on multiple GPUs, I think we need to split weight_shapes, so each GPU can tune all batch sizes and save complete config.
If splitting batch sizes, each runs the same shape with different batch sizes, but save_config does not support incremental saving.

## Checklist

- [x] Format your code according to the [Code Formatting with Pre-Commit](https://docs.sglang.ai/references/contribution_guide.html#code-formatting-with-pre-commit).
- [ ] Add unit tests as outlined in the [Running Unit Tests](https://docs.sglang.ai/references/contribution_guide.html#running-unit-tests-adding-to-ci).
- [ ] Update documentation / docstrings / example tutorials as needed, according to [Writing Documentation](https://docs.sglang.ai/references/contribution_guide.html#writing-documentation-running-docs-ci).
- [ ] Provide throughput / latency benchmark results and accuracy evaluation results as needed, according to [Benchmark and Profiling](https://docs.sglang.ai/references/benchmark_and_profiling.html) and [Accuracy Results](https://docs.sglang.ai/references/accuracy_evaluation.html).
- [ ] For reviewers: If you haven't made any contributions to this PR and are only assisting with merging the main branch, please remove yourself as a co-author when merging the PR.
- [x] Please feel free to join our Slack channel at https://slack.sglang.ai to discuss your PR.

### CODE REVIEW

**Comment by xihuai18** [comment]
Can you upload the tuned config?

**Comment by lambert0312** [comment]
> Can you upload the tuned config?

I can upload a800 tuned config

File: benchmark/kernels/quantization/tuning_block_wise_w8a8.py [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+from datetime import datetime
+from typing import Any, Dict, List
+
+import torch
+import triton
+from tqdm import tqdm
+
+mp.set_start_method("spawn", force=True)
+
+from sglang.srt.layers.quantization.fp8_kernel import (
```
Comment (lambert0312) [inline benchmark/kernels/quantization/tuning_block_wise_w8a8.py]: Why not use int8_kernel?

**Review by lambert0312 — COMMENTED** [review]
LGTM

**Comment by coolhok** [comment]
@HandH1998 Please help review the code

**Comment by HandH1998** [comment]
@coolhok Thanks for your nice work! But it seems that https://github.com/sgl-project/sglang/pull/3922 also did the same work and has been merged. If you find there is something different between the two PRs, please let me know.

**Comment by coolhok** [comment]
> @coolhok Thanks for your nice work! But it seems that #3922 also did the same work and has been merged. If you find there is something different between the two PRs, please let me know.

thanks,https://github.com/sgl-project/sglang/pull/3922 implementation is more elegant than mine,I will close it.
